src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -278,7 +278,7 @@ public DefaultWatch3DViewModel(NodeModel model, Watch3DViewModelStartupParams pa
 
             Name = Resources.BackgroundPreviewDefaultName;
             isGridVisible = parameters.Preferences.IsBackgroundGridVisible;
-            active = parameters.Preferences.IsBackgroundPreviewActive;
+            active = parameters.Preferences.GetIsBackgroundPreviewActive("IsBackgroundPreviewActive");
```

Comment:
Whats up with this?

Reply:
The background preview now have multiples states states by name. The obsoleted one was forwarded to this.

Reply:
theres no better API for this where you don't need to use a string everywhere?

Reply:
I will check.